README.md

```diff
@@ -1,38 +1,63 @@
+Service Binding Operator 1.0 was released at the end of 2021 and the Service Binding Spec 1.0 was released at the beginning of 2022. Although there have been some early efforts to support the concept of Provisioned Service, the adoption of the Service Binding specification is still in the bootstrap stage. There are still multiple service providers that have not yet implemented the provisioned service concept leaving application developers wondering how they can start implementing the standard on their side. To cover that transition the Service Binding specification also introduces the concept of [direct secret reference](https://github.com/servicebinding/spec#direct-secret-reference), which allows SEDs to be created manually by an administrator. As service providers implement the Service Binding specification, developers can start implementing their workloads to read data from these SEDs and be ready for the automation that will be available in the near future.
+
+With SEDs, an administrator can define bindable secrets that describe a service connection even if such services is not represented in the Kuberentes cluster. Examples of this type of services are services that are provisioned directly in public cloud providers. For service providers that want to implement Provisioned Services, the SEDs will serve as a guide for the type and format of data that workloads expect from their services when using Service Bindings. On the other hand, the application providers will be able to use SEDs to bind to services that are not even repersented with Kubernetes custome resources. 
 
 # SEDs as helm charts
-This project will deliver SEDs as helm charts. The requirements of the helm charts are as follows:
-1. Chart delivers a secret the follows the service binding spec.
-1. Chart has a test that can verify the health of the SED being defined.
+This project will deliver SEDs as Helm charts. The requirements of the Helm charts are as follows:
+1. Chart delivers a secret that follows the Service Binding specification for a specific cloud service.
+1. Chart has a test that can verify the health of the service represented by the SED being defined. See [Helm Chart Tests](https://helm.sh/docs/topics/chart_tests/) for more details and examples on how to implement a Helm test.
+
+
+NOTE:
+
+The aim of this repo is not to create a Helm chart repository, but to create an environment for peer review of SED Helm charts. After the chart is developed it will be maintained here, but the releases of the chart will happen on Helm chart repositories such as the [OpenShift Helm Chart Repository](https://github.com/openshift-helm-charts/charts). The OpenShift Helm Chart Repository is not only a Helm repository but also a Red Hat Helm chart certification flow that verifies whether your charts can run properly on the OpenShift Container Platform (OCP) clusters.
```

Comment:
```suggestion
This repository is not intended to create a Helm chart repository. The aim of this repository is to create an environment for the peer review of SED Helm charts. After the chart is developed, it will be maintained here. But the releases of the chart will happen on Helm chart repositories such as the OpenShift Helm Chart Repository. The OpenShift Helm Chart Repository is both a Helm repository and a Red Hat Helm chart certification flow that verifies whether your charts can run properly on OpenShift Container Platform (OCP) clusters.
```